src/createNoOpClient.ts

```diff
@@ -0,0 +1,15 @@
+import MetricsClient from './MetricsClient';
+
+/**
+ * Creates a no-op Datadog client
+ *
+ * This is useful if your application expects a MetricsClient
+ * but you'd like to disable std out in some environments
```

Comment:
stdout may be too specific here, since this should work in place of a StatsD client too

Reply:
good point